lib/v2/zhubai/index.js

```diff
@@ -0,0 +1,28 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const limit = 20;
```

Comment:
Consider allowing user changing this number using `ctx.query.limit`
https://docs.rsshub.app/parameter.html#tiao-shu-xian-zhi